src/main/java/org/minions/devfund/noemiguzman/battleship/Ocean.java

```diff
@@ -0,0 +1,187 @@
+package org.minions.devfund.noemiguzman.battleship;
+
+/**
+ * Ocean class.
+ */
+public class Ocean {
+    private static final int SIZE_OCEAN = 20;
+    private static final int NRO_SHIPS = 13;
+    private Ship[][] ships = new Ship[SIZE_OCEAN][SIZE_OCEAN];
+    private int shotsFired;
+    private int hitCount;
+    private int shipsSunk;
+
+    /**
+     * Creates an empty ocean (fills the ships array with a bunch of EmptySea instances).
+     * Also initializes any game variables, such as how many shots have been fired.
+     */
+    public Ocean() {
+        shotsFired = 0;
+        hitCount = 0;
+        shipsSunk = 0;
```

Comment:
Redundant initialization for integer attributes of class.
They are 0 by default.